modules/gathering-data-specific-features.adoc

```diff
@@ -44,6 +44,9 @@ ifndef::openshift-origin[]
 |`registry.redhat.io/openshift4/ose-cluster-logging-operator`
 |Data collection for OpenShift Logging.
 
+|`registry.redhat.io/openshift4/ose-local-storage-mustgather-rhel8`
+|Data collection for Local Storage Operator.
```

Comment:
@jsafrane Is there an upstream version of this image for OKD? Most of the items in this list currently also list a relevant image for OKD (in the below table).

Reply:
@bergerhoffer, I must be blind, but I don't see any "list of relevant images" for OKD below.

Reply:
Hi @jsafrane - @bergerhoffer is referring to the conditional table for OKD in lines 54-81 of this `modules/gathering-data-specific-features.adoc` file. The OKD table doesn't render in the preview build because it builds OCP (enterprise) docs only. I poked around quay.io but didn't see anything there for `ose-local-storage-mustgather-rhel8` or similar.

Reply:
Thanks for the pointer! I was looking at the generated html only.

I added quay.io/openshift/origin-local-storage-mustgather to the OKD part.

Reply:
Thanks @bobfuru for explaining, and thanks @jsafrane for the update! This LGTM now